Fix two bugs in master-setup.sh

jackgr opened the pull request:
Couple of bug fixes to make the script run correctly when it's run locally and when there's not already a cluster-insight replication controller running on the master.

EranGabber commented on install/master-setup.sh:
```diff
@@ -91,15 +91,20 @@ if [[ "${MASTER_ID}" != "" ]]; then
 fi
 
 # stop the Cluster-Insight minion collectors.
```
Please change the comment to:
# stop the Cluster-Insight replication controller (and its minions) if it is running.

jackgr replied:
Done. See comment on github.


Jack Greenfield | Google

On Tue, Jun 9, 2015 at 12:55 PM, Eran Gabber <notifications@github.com>
wrote:

> In install/master-setup.sh
> <https://github.com/google/cluster-insight/pull/79#discussion_r32056117>:
>
> > @@ -91,15 +91,20 @@ if [[ "${MASTER_ID}" != "" ]]; then
> >  fi
> >
> >  # stop the Cluster-Insight minion collectors.
>
> Please change the comment to:
> stop the Cluster-Insight replication controller (and its minions) if it is
> running.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cluster-insight/pull/79/files#r32056117>.
>

EranGabber commented on install/master-setup.sh:
```diff
@@ -91,15 +91,20 @@ if [[ "${MASTER_ID}" != "" ]]; then
-echo "stop the Cluster-Insight replication controller and minions"
-exec_command kubectl stop -f ${REP_CONTROLLER}
+rc_output="$(kubectl get rc | fgrep ${IMAGE})"
+if [[ -n $rc_output ]] ; then
```
You have to be careful when rc_output is empty. You may get a syntax error if it is empty.
Please change the condition to:
if [[ -n "${rc_output}" ]] ; then

EranGabber commented:
Please address my comments and ping me.

jackgr commented:
I've addressed your comments. Note that quoting is not necessary for -n, since it deals correctly with unset and null variables by design, but I've quoted the expression, anyway. On the comment, I took it verbatim, except for the parenthetical expression "(and it minions)" since replication controllers don't have minions - they have pods. If you want to add that back in or further revise the comment, I'd suggest that you do it after this PR is merged.
